src/js/settings.ts

```diff
@@ -151,7 +151,8 @@ export default {
     flipPieces: localstorageprop('settings.otb.flipPieces', false),
     useSymmetric: localstorageprop('settings.otb.useSymmetric', false),
     variant: localstorageprop('settings.otb.variant', 'standard'),
-    availableVariants: offlineAvailableVariants
+    availableVariants: offlineAvailableVariants,
+    savedGames: localstorageprop('settings.otb.savedGames', {})
```

Comment:
I don't want to store all played games right now. And not with localstorage. Localstorage space is limited and if we do that, we'd want to add the ability to delete past games.

And I don't get why you do this, since you always use the same id, if I get it correctly (`1`)